### PR TITLE
feat: update velero metadata with warning

### PR DESCRIPTION
**Pull request by takirala**
**What problem does this PR solve?**:

Add a warning in velero metadata.yaml about the **NEWLY** added ceph dependency.

**Which issue(s) does this PR fix?**:

https://d2iq.atlassian.net/browse/D2IQ-93534

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
- velero now depends on `rook-ceph` and `rook-ceph-cluster`
```

**If the PR adds a version bump, does it add a breaking change in License**:

- [ ] No License Change (or NA).

### CODE REVIEW

File: services/velero/metadata.yaml [hunk 1]
```diff
@@ -14,6 +14,11 @@ overview: |-
   Velero is an open source tool for safely backing up and restoring resources in a Kubernetes cluster, performing disaster recovery, and migrating resources and persistent volumes to another Kubernetes cluster.
 
   Velero offers key data protection features, such as scheduled backups, retention schedules, and pre- or post-backup hooks for custom actions. The tool can help protect data stored in persistent volumes and makes your entire Kubernetes cluster more resilient.
+  
+  ## Attention Grabbing Title Here
```
Comment (takirala) [inline services/velero/metadata.yaml:18]: Ping @charIesdu @D2iQdocs Can you ptal and suggest the content here as this will be visible to user in DKP UI.

Comment (takirala) [inline services/velero/metadata.yaml:18]: Ping @mstruebing : PTAL and suggest if there is any markdown magic we can do here in context of https://d2iq.atlassian.net/browse/D2IQ-93534

Comment (mstruebing) [reply services/velero/metadata.yaml:18]: Sorry @takirala but we can not do anything special here that isn't regular markdown. 
If we want to, we would need to implement something I'm not sure we have the bandwidth for.

Comment (takirala) [reply services/velero/metadata.yaml:18]: Sounds good - i didn't mean to infer we should implement something new. Just wondering if we already have something (markdown tags or anything else) that we can use for a `WARNING` note.

**Comment by takirala** [comment]
Ping @UXAndrew can you also please advise here!

**Comment by UXAndrew** [comment]
I responded on the Jira ticket @takirala LMK if you'd like to discuss further.

File: services/velero/metadata.yaml [hunk 2]
```diff
@@ -14,6 +14,11 @@ overview: |-
   Velero is an open source tool for safely backing up and restoring resources in a Kubernetes cluster, performing disaster recovery, and migrating resources and persistent volumes to another Kubernetes cluster.
 
   Velero offers key data protection features, such as scheduled backups, retention schedules, and pre- or post-backup hooks for custom actions. The tool can help protect data stored in persistent volumes and makes your entire Kubernetes cluster more resilient.
+  
+  ## Dependency on Rook Ceph Cluster
+  In previous versions of DKP (2.3.x), Velero was configured to deploy a standalone MinIO upon demand. In DKP 2.4.x and above, MinIO is replaced with Rook Ceph Cluster as the default Backend.
+  If you plan to use Velero with Ceph, then you need to explicitly install Rook Ceph Cluster to use velero (E.g.: on-prem clusters). It is recommended to use a storage backend that is outside your DKP cluster for production environments.
```
[the remark this answers is not in the document]
Comment (takirala) [reply services/velero/metadata.yaml:20]: I updated with explanation. I think the reason is self-explanatory - i copied the structure from https://docs.d2iq.com/dkp/2.3/backup-and-restore where we just say 
```
For production use-cases, D2iQ advises to provide an external storage class to use with [MinIO](https://docs.min.io/).
```
but i updated the description with some context. lmk if this makes sense (or not).

[the remark this answers is not in the document]
Comment (charIesdu) [reply services/velero/metadata.yaml:20]: LGTM

**Review by charIesdu — APPROVED** [review]

**Review by cbuto — APPROVED** [review]

File: services/velero/metadata.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (takirala) [reply services/velero/metadata.yaml]: lol i totally pulled in your suggestion (even added attribution for you as co-author in commits) but must have lost it when i was doing rebasing.

fixed in [`bb1462c` (#732)](https://github.com/mesosphere/kommander-applications/pull/732/commits/bb1462c8337b180b1166f11bc282cf1b8563dbeb) for real.

**Review by cwyl02 — APPROVED** [review]

**Pull request merged**